src/Common/tests/XunitTraitsDiscoverers/PlatformID.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    [Flags]
+    public enum PlatformID
+    {
+        Windows = 1,
+        Linux = 2,
+        OSX = 4,
+        Any = 8
```

Comment:
Just is correct from the naming scheme we do use plural for flags enums. Also Why wouldn't Any be initialized to PlatformID.Windows | PlatformID.Linux | PlatformID.OSX so it is explicit that is what it represents, it would also enable folks to use the bitmask instead of special casing it.